### PR TITLE
Enable Org Pro Dashboard View

### PR DESCRIPTION
**Pull request by Zhao-Andy**
This PR enables the org pro dashboard view. I decided to leave refactoring and making more data available for the next PR.

### CODE REVIEW

File: app/facades/dashboard.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Dashboard
+  attr_accessor :user
```
Comment (lightalloy) [inline app/facades/dashboard.rb]: Probably `attr_reader` would be enough here.

File: app/facades/pro.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
+class Pro < Dashboard
+  def initialize(user)
+    super
+  end
+
+  def user_article_ids
+    user.articles.pluck(:id)
```
Comment (lightalloy) [inline app/facades/pro.rb]: I suggest memoizing like `@user_articlle_ids ||= user.articles.pluck(:id)`. The method is called multiple times, so with memoization, the values won't need to be arranged multiple times.

File: app/facades/pro.rb [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  def this_week_reactions
+    ChartDecorator.decorate(Reaction.where(reactable_id: user_article_ids, reactable_type: "Article").where("created_at > ?", 1.week.ago).order("created_at ASC"))
+  end
+
+  def this_week_reactions_count
+    this_week_reactions.size
+  end
+
+  def last_week_reactions_count
+    Reaction.where(reactable_id: user_article_ids, reactable_type: "Article").where("created_at > ? AND created_at < ?", 2.weeks.ago, 1.week.ago).size
```
Comment (lightalloy) [inline app/facades/pro.rb]: It's tempting to create private methods like `reactions_scope` and `comments_scope` for the repeating parts, just a suggestion.

File: app/facades/pro.rb (hunk 2 above)
Comment (Zhao-Andy) [reply app/facades/pro.rb]: TIL!

File: app/facades/pro.rb (hunk 3 above)
Comment (Zhao-Andy) [reply app/facades/pro.rb]: Yeah, I think I'll do that. There's definitely a better way to organize the mass amount of data.

File: app/facades/dashboard.rb (hunk 1 above)
Comment (Zhao-Andy) [reply app/facades/dashboard.rb]: 👍

**Pull request merged**